topydo/ui/TodoListWidget.py

```diff
@@ -199,6 +203,17 @@ def mouse_event(self, p_size, p_event, p_button, p_column, p_row, p_focus):
     def selectable(self):
         return True
 
+    def _append_pending_todos(self):
+        try:
+            todo = self.listbox.focus.todo
+            todo_id = str(self.view.todolist.number(todo))
+            result = urwid.emit_signal(self, 'append_pending_todos', todo_id)
+            attr_spec = {None: markup(todo, result)}
```

Comment:
Please add methods `mark` / `unmark` to `TodoWidget`, and let the TodoWidget figure out how it should render itself. This is not the responsibility of the TodoListWidget. So the `markup` function stays internal.